src/components/FindMovie/FindMovie.tsx

```diff
@@ -1,10 +1,67 @@
-import React from 'react';
+import React, { useCallback, useState } from 'react';
+import classNames from 'classnames';
+import { getMovie } from '../../api';
+import { MovieCard } from '../MovieCard';
 import './FindMovie.scss';
+import { Movie } from '../../types/Movie';
+
+type Props = {
+  onAddMovie: (movie: Movie) => void;
+};
+
+export const FindMovie: React.FC<Props> = ({ onAddMovie }) => {
+  const [movie, setMovie] = useState<Movie | null>(null);
+  const [query, setQuery] = useState('');
+  const [isError, setIsError] = useState(false);
+  const [isLoading, setIsLoading] = useState(false);
+
+  const handleSubmit = useCallback(
+    async (event: React.FormEvent) => {
+      event.preventDefault();
+      setIsLoading(true);
+      const result = await getMovie(query);
+
+      setIsLoading(false);
+
+      if ('Error' in result) {
+        setIsError(true);
+      } else {
+        setMovie({
+          title: result.Title,
+          description: result.Plot,
+          imgUrl: result.Poster === 'N/A'
```

Comment:
better to put N/A in a variable at the top of the files after import, so it's easy to change